docs/PATCHEXPLAIN.md

```diff
@@ -139,6 +139,10 @@ Below is a rundown of the main logic that OpenCore Legacy Patcher uses to gain n
   * Reason: Enables Content Caching when using VMM spoofing
   * Logic: Adds args to NVRAM
   * Models: Any model using VMM spoofing
+* `-amfi_get_out_of_my_way=0x1`
```

Comment:
1. Typo, arguments in macOS only start with a `-` if they have no suffix number
2. OCLP doesn't use this arg, it uses `amfi=0x80`
Reference:
https://github.com/dortania/OpenCore-Legacy-Patcher/blob/0.6.4/resources/amfi_detect.py